doc/conf.py

```diff
@@ -263,5 +263,5 @@ def setup(app):
 
 config_block = {
     'varnish3': 'Varnish 3',
-    'varnish4': 'Varnish 4'
+    'varnish4': 'Varnish 4 & 5'
```

Comment:
should the key be renamed to ``varnish`` too (changing the doc too) as it is not only about varnish 4 anymore ?